spark/v3.2/spark-extensions/src/main/scala/org/apache/spark/sql/execution/datasources/v2/OptimizeMetadataOnlyDeleteFromTable.scala

```diff
@@ -19,6 +19,7 @@
 
 package org.apache.spark.sql.execution.datasources.v2
 
+import org.apache.spark.internal.Logging
```

Comment:
We should not use Spark's internal logging. Can you convert this to use SLF4J instead?

Reply:
Done.  

Interestingly, turns out this object was actually extending 'logging' trait transitively and so already had 'log' variable in scope.  Named the SLF4J logger used here as 'logger'.